imports/plugins/included/sitemap-generator/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,19 @@
+extend type Mutation {
+  "Generate sitemap documents"
+  generateSitemaps(input: GenerateSitemapsInput): GenerateSitemapsPayload
```

Comment:
Change to `GenerateSitemapsPayload!` to indicate that the response will always be the payload object, never `null`.